### DISCUSSION TITLE
user-guide/

### DISCUSSION THREAD

**Discussion by giscus** [zotero-chinese/wiki#77] [文档评论区]
# user-guide/

在 Zotero 使用过程中遇到问题，先上 Zotero 百科全书搜索一下！ 本文档由 Zotero Chinese (https://github.com/zotero-chinese) 团队维护。有任何问题，欢迎在文档下方留言！ 本文档地址：https://zotero-chinese.com/ (https://zotero-chinese.co...

https://zotero-chinese.com/user-guide/

**Comment by Komorebi-Mason** [comment]
功在当代，利在千秋！！！

**Comment by zhaochengye** [comment]
承前启后，继往开来！

**Comment by Hughlau436** [comment]
求更新！

**Comment by cocoper** [comment]
感谢！

**Comment by leoatchina** [comment]
林同学V5

**Comment by Donnacod** [comment]
感谢

**Comment by yiranzhimo** [comment]
很厉害，谢谢你们🙏！

**Comment by ggq680** [comment]
好棒！让zotero变得更好

**Comment by maohy915** [comment]
thanks for your working, best love from beijing

**Comment by ChicagoXI** [comment]
请问如何从一个文献链接回到它所在的分类文档里呢，有时候知道自己想看什么文献，也能从“我的文库”里找到，但是如何知道它是属于哪个分类文档呢

**Reply by northword** [reply]
按住ctrl再单击文献，左侧的分类里会高亮当前条目所在的分类。
zotero 7的话，右侧会列出当前条目所在的所有分类。

**Reply by northword** [reply]
按住ctrl再单击文献，左侧的分类里会高亮当前条目所在的分类。
zotero 7的话，右侧会列出当前条目所在的所有分类。

**Reply by ChicagoXI** [reply]
谢谢，很管用的技能！

**Reply by permia** [reply]
zotero 7.0.30 (64-bit),   鼠标单击选中文献标题，再按下ctrl 就可以高亮分类。

**Comment by zyh1231** [comment]
请问为什么我的connect的扩展选项-Advanced-会出现Report Error？我也没办法抓取中文的和英文PDF

**Reply by zyh1231** [reply]
[JavaScript Error: "Translator called selectItems with no items
Error: Translator called selectItems with no items

**Comment by boven-xx** [comment]
好！好！好！

**Comment by fisherchase** [comment]
这个能否下载到本地阅读啊？（epub或者pdf格式）

**Reply by northword** [reply]
目前不能，因为文档还在不断完善中，短时间内不计划提供 pdf 或 epub 版本。

**Comment by kakaluotek** [comment]
好！！！

**Comment by yxyyxx** [comment]
请问导入文献后为什么会出现文献中的某一个figure被蓝灰色色块遮挡

**Reply by yxyyxx** [reply]
使用常见问题中关于“PDF存在黑色遮罩”的解决办法，并没有解决问题

**Comment by xianchengyue** [comment]
请问同一个条目下的多个pdf文件怎么调整顺序呢？，有时候先添加正文，后添加附件，预览的时候出现的就是附件而不是正文

**Reply by northword** [reply]
参考：https://github.com/orgs/zotero-chinese/discussions/167

**Comment by linsonpi** [comment]
为什么按照流程一步步完成了之后，放在坚果云文件夹内的pdf文件，同步后并不显示在zotero里啊

**Comment by KeZhoy3** [comment]
智齿！

**Comment by zykkkai** [comment]
伟大！

**Comment by megumiXzK** [comment]
感谢分享，另外想问一下是否有从注释跳转/链接到笔记的方案。

**Comment by gracy-yang** [comment]
请问下，我的中文文献拖入zoreto之后，名字都显示为英文，这个怎么调整？找了半天原因也没找到，zoreto7

**Comment by Marigolddd** [comment]
感恩

**Comment by Hazelnut-s** [comment]
请问为什么把高亮段落添加到条目笔记中会出现高亮颜色的代码呢？

**Comment by yubinghong-fish** [comment]
zotero右边条目信息栏里面有些信息我是不需要显示的，能否自定义隐藏一些信息，或者将这些信息自定义排序？

**Comment by ChicagoXI** [comment]
重装系统了，坚果云里的文献如何导入Zotero呢？

**Comment by GM-2024** [comment]
支持

**Comment by UApocalypse** [comment]
会有鸿蒙移动端吗？

**Comment by xub81** [comment]
感谢大佬！

**Comment by lucerna-gelu** [comment]
有人和我一样觉得内置的阅读器对于中文的渲染不太令人满意吗，和其他任何pdf阅读器比起来都有字体发虚的情况，请问这个问题有没有什么解决方法？

**Reply by JayHoris** [reply]
不只是中文，我发现升级到zotero7之后，英文阅读也出现了文本似乎显示不清晰的问题，一种类似于单词“断墨”的感觉。需要将文本放大很多，这种问题才会消失。同样的pdf文本在其他阅读器中就不会出现这种情况。感觉像是dpi的问题，我的是4k屏幕，不知道是否因为zotero7的阅读器不太适配4k屏幕的dpi。

**Comment by tongn9** [comment]
非常感谢你们的付出！！！

**Comment by YuHaoyang2002** [comment]
您好，请问我设置过引用人数最多为3，否则是输出等或etal 但是为什么我在样式编辑器里面都显示人名1，etal，但是word引用就人名1，人名2，etal

**Comment by v3xyan** [comment]
在citation中已经增加了页码，但在参考文献中没有显示出来，该怎么办？
我看知乎上这个人也碰到了同样的问题，但是没有答案。
https://zhuanlan.zhihu.com/p/345447630

**Comment by wallenwong** [comment]
请问这款软件的名称Zotero的音标是什么，英文怎么读？

**Comment by left99** [comment]
感谢大佬们的付出！有个小疑问，请问标题可以换行显示吗，谢谢~

**Comment by chdliu1974** [comment]
接入deepseek后应用askpdf 没有结果，提示如下，怎么解决？

🪐AskPDF execution error: _0x254a78[_0x4f4dac(...)][_0x4f4dac(...)](...) is undefined

Meet.Zotero.getRelatedText(Meet.Global.input || "概括本文")

---
Please check the code inside the command tag.

**Reply by chdliu1974** [reply]
在哪里检查？检查什么？什么样的是正确的？谢谢

**Comment by Ooshisen** [comment]
感谢你们，伟大！

**Comment by Cnc-2991** [comment]
您好，我们有用户使用这个产品，想做一些开发，有能开发Zotero产品的朋友请尽快联系我
VX：Cnc2991

**Comment by MYX0910** [comment]
好好好！

**Comment by yulianwete** [comment]
非常感谢有这么好的软件。

**Comment by Wojiuxihuanhaha** [comment]
感谢！有一个问题想请教，我的zotero中显示加载项Microsoft word 未安装，但是点击安装加载项Microsoft word没有任何反应，回到word中同样没有出现zotero，在word的共用模版和加载项中也没有zotero，请问这是为什么呀？能怎么解决？谢谢

**Comment by RENSHUHUA9** [comment]
为什么我的翻译 还是显示英文原文呢？ 有没有碰到过类似问题的，求助

**Comment by boven-xx** [comment]
挂梯子试试




博文_
***@***.***

华东师范大学

**Comment by lixiaohong0323** [comment]
请问为什么zotero文中插入文献是好的，是APA格式，但参考文献表的格式是错误的，不是APA格式了。

**Comment by Krones-biotech** [comment]
蒸蒸日上！

**Comment by liuyu268** [comment]
太厉害了

**Comment by champersnova** [comment]
十分感谢！！！

**Comment by TanKurcheng** [comment]
您好，请问我原先使用zotero 6，用better note在pdf里面做了很多笔记卡片，现在如果更新到zotero 7的话，better note的资料会被保留吗？重要的资料几乎都在better note做的卡片里面。

**Reply by TanKurcheng** [reply]
因为是插件，不同于zotero原生的程序，有点担心插件里的东西不会被保留，不懂软件方面的知识，望赐教

**Comment by v3xyan** [comment]
这个插件我没用过，建议你可以用另外一台电脑试一下。
理论上软件升版应该不会删除原来的内容。

**Reply by TanKurcheng** [reply]
嗯好，谢谢！

**Comment by boven-xx** [comment]
请问如何添加国家标注，例如引用HJ 915-2017

**Comment by ChiuUwU** [comment]
请问为什么我的zotero connector点开浏览器的扩展选项后，没有内容就是一个灰色的页面，点击之后zotero也没有反应也不是pdf，以前就是可以正常点击爬取pdf

**Comment by Akmangday** [comment]
太爱了，谁懂我在别处都没查到的细节在这里一下就查出来了，新手入坑和老手查漏补缺都超级好用，比起网上的零碎笔记、或看似系统但又不够全面且不方便专门查阅的视频教程，甚至是官方那些缺乏统一整理的更新日志和较为老旧的使用手册（更别说还有语言隔阂），在这里入手学习的性价比可以说是最高了，要想在最短时间内学到最需要的全面内容，这里绝对是最值得推荐的了

**Comment by ck1005974** [comment]
您好，我使用的版本是zotero7，软件并未更新，电脑文件zotero里还有一些插件的文件，但是所有的插件（包括插件市场）都没有了。请问这是为什么？有解决办法吗？谢谢！

**Comment by DrMaXYC** [comment]
牛哇牛哇
